codegen/README.md

```diff
@@ -4,6 +4,14 @@ Smithy code generators for Go.
 
 **WARNING: All interfaces are subject to change.**
 
+## Setup
+
+1. Install Java 11. If you have multiple versions of Java installed on OSX, use `export JAVA_HOME=`/usr/libexec/java_home -v 11``. **Java 14 is not compatible with Grade 5.x**
+2. Install Go (follow directions for your platform)
+3. Use `./gradlew` to automatically install the correct gradle version. **`brew install gradle` will install Gradle 6.x which is not compatible.**
+4. `./gradlew test` to run the basic tests.
+5. `cd smithy-go-codegen-test; ../gradlew build` to run the codegen tests
```

Comment:
nit `gradlew` is nested under the `codegen` folder.  Probably want to callout htat these `gradlew` commands are executed from that working directly.

Reply:
This is the README that's inside of the codgen folder, so I gave the commands relative to the `codegen` folder

Reply:
Ah thanks, didn't realize that!